### PR TITLE
GW BatchWrite sizes

### PR DESCRIPTION
**Pull request by bipin-rudder**
**Fixes** # (*issue*)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: gateway/gateway.go [hunk 1]
```diff
@@ -237,6 +238,11 @@ func (gateway *HandleT) dbWriterWorkerProcess(process int) {
 		} else {
 			gateway.jobsDB.Store(jobList)
 		}
+		batchPayloadSize := 0
+		for _, req := range jobList {
+			batchPayloadSize += len(req.EventPayload)
+		}
+		gateway.batchPayloadSizeStat.Count(batchPayloadSize)
```
Comment (SumanthPuram) [inline gateway/gateway.go:245]: Here we are measuring DB write size, instead, let's measure the event size. 

Either size of the HTTP request payload or single event size. Once we have that, we can derive these values using some heuristics.

Comment (bipin-rudder) [reply gateway/gateway.go]: done

File: gateway/gateway.go [hunk 2]
```diff
@@ -357,6 +358,8 @@ func (gateway *HandleT) userWebRequestWorkerProcess(userWebRequestWorker *userWe
 				misc.IncrementMapByKey(sourceFailStats, sourceTag, 1)
 				continue
 			}
+			gateway.logger.Debugf("event size from http request : %d", len(body))
```
Comment (chandumlg) [inline gateway/gateway.go:361]: Remove this log. This will make it very difficult to view the logs.

**Review by chandumlg — APPROVED** [review]

File: gateway/gateway.go (hunk 2 above)
Comment (bipin-rudder) [reply gateway/gateway.go]: done

**Review by chandumlg — APPROVED** [review]

**Pull request merged**